### DISCUSSION TITLE
Coordinates - longitude/latitude VS x/y/z

### DISCUSSION THREAD

**Discussion by clemencestr** [iTowns/itowns#1730] [Q&A] [answered]
Hi !  

In the constructor of src/Core/Geographic/Coordinates.js, we have:

- x = longitude
- y = latitude
-  z = altitude

In the constructor of Coordinates, the documentation states that we can input x or longitude value for the same parameter. It is the same for y and z (latitude and  altitude). However, there are not supposed to have the same value, one is in meter the other in degree. There should be a conversion but I have not seen any. Is that normal ?

Thanks for your advises, 

Clemence

**Comment by gchoqueux** [comment, marked as answer]
Hello Clemence

The `EPSG` indicates unit value (meter or degree) for `v0` and `v1`.
For `v3` is always in meter for altitude.

**Reply by clemencestr** [reply]
Ok thank you very much
